README.md

```diff
@@ -92,6 +92,49 @@ kubectl apply -f ./artifacts/operator-rbac.yaml
 kubectl apply -f ./artifacts/operator.yaml
 ```
 
+## Running in-cluster, using Google Compute Engine for the exit node using helm
```

Comment:
Can you remove this section and change it to running in-cluster with options provided by helm?

Reply:
I mis-read this, it's fine, but do you need to tell people to to first create a project and init / auth to Google Cloud?

Reply:
Yes in order to get the credential and the service account you have to have authenticated your account and initialised gcloud with the project and account you want to use, as gcloud can be used with multiple accounts and projects by switching configs. This also makes sure that even when gcloud is initialised, users of inlets-operator do not accidentally create resources on projects they do not intend to by giving them a heads up about initialising gcloud.

Reply:
That's good info, I think it needs too be in the README in this section